src/Hazelcast.Net/HazelcastClient.Objects.cs

```diff
@@ -25,6 +26,12 @@ namespace Hazelcast
     internal partial class HazelcastClient // Distributed Objects
     {
         private readonly ISequence<long> _lockReferenceIdSequence = new Int64Sequence();
+        private ICP _cp;
+
+        /// <summary>
+        /// Gets the CP subsystem.
+        /// </summary>
+        public ICP CP => _cp ??= new CP.CP(Cluster, _distributedOjects);
```

Comment:
All other clients name this as `CPSubsystem`, can we use the same naming here as well?

Reply:
yes, renaming everywhere.